Proposals/Glyph_Extension_Axis/ProposalSummary.md

```diff
@@ -0,0 +1,69 @@
+# Proposal Summary Form
+
+## Administrative Information
+**Proposers name:** Sahar Afshar and José Solé
+
+**Vendor affiliation:** N/A
```

Comment:
Is there no vendor backing this yet?

Reply:
No, this is being done independent of any vendor affiliations.